dask/dataframe/io/delta/core.py

```diff
@@ -0,0 +1,289 @@
+            .to_pandas()
+        )
+
+    def read_delta_table(self, **kwargs):
+        """
+        Reads the list of parquet files in parallel
+        """
+        if len(self.pq_files) == 0:
+            raise RuntimeError("No Parquet files are available")
+        parts = [
+            delayed(self.read_delta_dataset)(f, **kwargs) for f in list(self.pq_files)
+        ]
+        return from_delayed(parts)
+
+
+def read_delta_table(
+    path,
+    version=None,
+    columns=None,
+    engine="auto",
+    checkpoint=None,
+    storage_options=None,
+    **kwargs,
+):
+    """
+    Read a Delta Table into a Dask DataFrame
+
+    This reads a list of Parquet files in delta table directory into a
+    Dask.dataframe.
+
+    Parameters
+    ----------
+    path: str
+        path of Delta table directory
+    version: int, default 0
+        DeltaTable Version, used for Time Travelling across the
+        different versions of the parquet datasets
+    columns: None or list(str)
+        Columns to load. If None, loads all.
+    engine : str, default 'auto'
+        Parquet reader library to use. Options include: 'auto', 'fastparquet',
+        'pyarrow', 'pyarrow-dataset', and 'pyarrow-legacy'. Defaults to 'auto',
+        which selects the FastParquetEngine if fastparquet is installed (and
+        ArrowDatasetEngine otherwise).  If 'pyarrow' or 'pyarrow-dataset' is
+        specified, the ArrowDatasetEngine (which leverages the pyarrow.dataset
+        API) will be used. If 'pyarrow-legacy' is specified, ArrowLegacyEngine
+        will be used (which leverages the pyarrow.parquet.ParquetDataset API).
+        NOTE: The 'pyarrow-legacy' option (ArrowLegacyEngine) is deprecated
+        for pyarrow>=5.
+    checkpoint: int, default None
+        DeltaTable protocol aggregates the json delta logs and creates parquet
+        checkpoint for every 10 versions.This will save some IO.
+        for example:
+            if you want to read 106th version of dataset, No need to read all
+            the all the 106 json files (which isIO heavy), instead read 100th
+            parquet checkpoint which contains all the logs of 100 versions in single
+            parquet file and now read the extra 6 json files to match the given
+            106th version.
+            (so IO reduced from 106 to 7)
+    storage_options : dict, default None
+        Key/value pairs to be passed on to the file-system backend, if any.
+    kwargs: dict,optional
+        Some most used parameters can be passed here are:
+        1. schema
+        2. filter
+
+        schema : pyarrow.Schema
+            Used to maintain schema evolution in deltatable.
+            delta protocol stores the schema string in the json log files which is converted
+             into pyarrow.Schema and used for schema evolution (refer delta/utils.py).
+            i.e Based on particular version, some columns can be shown or not shown.
+        filter: pyarrow.dataset.Expression
```

Comment:
This is not the kind of filter that is used for the normal parquet read functions

Reply:
Got it, Thanks for letting me know about this, now changed the code to accept filters like ` [("x","==",1)]` similar to read_parquet